kegtab/src/main/java/org/kegbot/app/TapStatusFragment.java

```diff
@@ -256,6 +258,21 @@ public void onClick(View v) {
       description = tap.getDescription();
     }
 
+    // Find ABV and IBU values
+    final String abv = String.valueOf(keg.getBeverage().getAbvPercent());
+    if(mCore.getConfiguration().getAbvVisibleWhenZero()) {
+      abvText.setText("ABV: " + abv + "%");
```

Comment:
I'd personally prefer it to read more naturally, like `6.7% ABV`

Reply:
I could agree with that, thoughts on also flipping the IBU display? I think having both labels as a prefix or suffix would look best vs. having one as 6.7% ABV and the other IBU: 50.